packages/eslint-plugin-actual/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "eslint-plugin-actual",
+  "private": true,
+  "main": "./lib/index.js",
+  "exports": "./lib/index.js",
+  "type": "commonjs",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "mocha tests --recursive"
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏any way we could port these to use `jest`? Would be cool to stick to just one type of test-runner in the entire repo.